comb_spec_searcher/comb_spec_searcher.py

```diff
@@ -496,16 +497,22 @@ def _log_status(self, start_time: float) -> None:
             round(time.time() - start_time, 2)
         )
         elaborate = time.time() - start_time > 100 * self.func_times["status"]
+
         status_start = time.time()
         status += self.status(elaborate=elaborate)
+
+        next_elaborate = max(
+            0, 100 * self.func_times["status"] - (time.time() - start_time)
+        )
```

Comment:
Should we add also the time to the next status update?
`status_update - (time.time() - status_start)`

Reply:
Not sure what you're referring to.

Reply:
Well currently it might say 0 seconds, even though the next status update is in 12 hours

Reply:
Oh, are you saying instead of having "0" in the max, it should have the time until the next status update?

Reply:
I changed this, and also made further improvements by always taking the status_update interval into account. So, if there is a status update every 10 seconds and the next elaborate update is calculated to take place in 45.2 seconds, it will just say 50 seconds.

Reply:
I like this a lot!